microservices/front-end/src/components/collaboration/CollaborationPage.tsx

```diff
@@ -72,20 +76,24 @@ function CollaborationPage() {
         <div className={classes.grid_question}>
           <CollabQuestionPage difficulty={difficulty as QuestionComplexity} />
         </div>
-
         <div className={classes.grid_editor}>
-          <Editor socket={socket} />
+          <Editor
+            socket={socket}
+            setStdout={setStdout}
+            setIsoutputLoading={setIsOutputLoading}
```

Comment:
Code style: inconsistent camelCase naming of variables

Expected: `setIsOutputLoading={setIsOutputLoading}` (note the "O" in "Output" for the left hand variable)